examples/js/controls/OrbitControls.js

```diff
@@ -116,7 +116,22 @@ THREE.OrbitControls = function ( object, domElement ) {
 
 		scope.update();
 
-		state = STATE.NONE;
+		scope.cancel();
```

Comment:
I'm not sure it makes sense to fire an `end` event in `reset()`. How about keeping `reset()` as it was and only focus on the event listeners?

Reply:
My feeling was that for every "start" event we should be guaranteed an "end" event when the orbit controls interaction stops, which is why I added an "end" dispatch only when the controls are in the middle of a drag. Without this (and as can happen now) it's possible to get two "start" events dispatched back to back.

Reply:
@Mugen87 the structure of the fix has changed a bit -- do you have any further thoughts on how this should be handled?